portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -116,6 +124,11 @@ public CommentBuilder sentimentScore(double sentimentScore) {
       return this;
     }
 
+    public CommentBuilder imageLabels(List<Label> imageLabels) {
+      this.imageLabels = imageLabels;
+      return this;
```

Comment:
Do you really need to return a reference to self here? Also it looks like a good use case for a javascript setter (not sure if those are encouraged):
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/set

Reply:
I'm returning the reference to self so I can chain the different functions in the builder. Also, I don't understand how to use the javascript setter in this case.

Reply:
It turns out setters are not encouraged anyway.